### PR TITLE
Use RDatasets

### PR DESCRIPTION
**Pull request by devmotion**
This PR supersedes https://github.com/juliangehring/Bootstrap.jl/pull/50.

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -1,27 +1,28 @@
 name = "Bootstrap"
 uuid = "e28b5b4c-05e8-5b66-bc03-6f0c0a0a06e0"
-authors = "Julian Gehring"
-version = "2.1.0"
 license = "MIT"
+authors = "Julian Gehring"
```
Comment (juliangehring) [inline Project.toml]: What is the effect of changing the order here?

File: Project.toml [hunk 2]
```diff
@@ -1,27 +1,28 @@
...
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 StatsBase = "2913bbd2-ae8a-5f71-8c99-4fb6c76f3a91"
 StatsModels = "3eaba693-59b7-5ba5-a881-562e759f1c8d"
 
 [compat]
-julia = "1"
 DataFrames = "≥ 0.19.0"
 GLM = "≥ 1.3.0"
 StatsModels = "≥ 0.6.0"
+julia = "1"
```
Comment (juliangehring) [inline Project.toml]: Does this make a difference if julia comes last or is this merely style?

File: test/test-bootsample-non-parametric.jl [hunk 3]
```diff
@@ -1,15 +1,18 @@
...
 using Statistics
 using Random
 
 using DataFrames
+using RDatasets
 using StatsBase
 
+## 'city' dataset
+const city = dataset("boot", "city")
+const citya = convert(Matrix, city)
```
Comment (juliangehring) [inline test/test-bootsample-non-parametric.jl]: Okay, makes sense to define both data sets in one place.

File: Project.toml (hunk 1 above)
Comment (devmotion) [reply Project.toml]: It doesn't have any effect. That's just the order `Pkg` creates as soon as you activate the project and add or remove a dependency (not sure anymore if that's needed at all or if activating it is already sufficient.

File: Project.toml (hunk 2 above)
Comment (devmotion) [reply Project.toml]: Same here.

**Pull request merged**